client/styles/GlobalStyle.js

```diff
@@ -18,6 +18,7 @@ const GlobalStyle = createGlobalStyle`
     a {
       color: var(--orange);
     }
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
we might have this doubled, I can pull mine out

Reply:
i think git automatically merged this, so we should be good!